src/data/pcodes.json

```diff
@@ -1,8 +1,5224 @@
+     "P2359" : {
+	"description" : ""
+     },
+     "P2360" : {
+	"description" : ""
+     },
+     "P2361" : {
+	"description" : ""
+     },
+     "P2362" : {
+	"description" : ""
+     },
+     "P2363" : {
+	"description" : ""
+     },
+     "P2364" : {
+	"description" : ""
+     },
+     "P2365" : {
+	"description" : ""
+     },
+     "P2366" : {
+	"description" : ""
+     },
+     "P2367" : {
+	"description" : ""
+     },
+     "P2368" : {
+	"description" : ""
+     },
+     "P2369" : {
+	"description" : ""
+     },
+     "P2370" : {
+	"description" : ""
+     },
+     "P2371" : {
+	"description" : ""
+     },
+     "P2372" : {
+	"description" : ""
+     },
+     "P2373" : {
+	"description" : ""
+     },
+     "P2374" : {
+	"description" : ""
+     },
+     "P2375" : {
+	"description" : ""
+     },
+     "P2376" : {
+	"description" : ""
+     },
+     "P2377" : {
+	"description" : ""
+     },
+     "P2378" : {
+	"description" : ""
+     },
+     "P2379" : {
+	"description" : ""
+     },
+     "P2380" : {
+	"description" : ""
+     },
+     "P2381" : {
+	"description" : ""
+     },
+     "P2382" : {
+	"description" : ""
+     },
+     "P2383" : {
+	"description" : ""
+     },
+     "P2384" : {
+	"description" : ""
+     },
+     "P2385" : {
+	"description" : ""
+     },
+     "P2386" : {
+	"description" : ""
+     },
+     "P2387" : {
+	"description" : ""
+     },
+     "P2388" : {
+	"description" : ""
+     },
+     "P2389" : {
+	"description" : ""
+     },
+     "P2391" : {
+	"description" : ""
+     },
+     "P2392" : {
+	"description" : ""
+     },
+     "P2393" : {
+	"description" : ""
+     },
+     "P2394" : {
+	"description" : ""
+     },
+     "P2400" : {
+	"description" : ""
+     },
+     "P2401" : {
+	"description" : ""
+     },
+     "P2402" : {
+	"description" : ""
+     },
+     "P2403" : {
+	"description" : ""
+     },
+     "P2404" : {
+	"description" : ""
+     },
+     "P2405" : {
+	"description" : ""
+     },
+     "P2406" : {
+	"description" : ""
+     },
+     "P2407" : {
+	"description" : ""
+     },
+     "P2408" : {
+	"description" : ""
+     },
+     "P2409" : {
+	"description" : ""
+     },
+     "P2410" : {
+	"description" : ""
+     },
+     "P2411" : {
+	"description" : ""
+     },
+     "P2412" : {
+	"description" : ""
+     },
+     "P2413" : {
+	"description" : ""
+     },
+     "P2414" : {
+	"description" : ""
+     },
+     "P2415" : {
+	"description" : ""
+     },
+     "P2416" : {
+	"description" : ""
+     },
+     "P2417" : {
+	"description" : ""
+     },
+     "P2418" : {
+	"description" : ""
+     },
+     "P2419" : {
+	"description" : ""
+     },
+     "P2420" : {
+	"description" : ""
+     },
+     "P2421" : {
+	"description" : ""
+     },
+     "P2422" : {
+	"description" : ""
+     },
+     "P2423" : {
+	"description" : ""
+     },
+     "P2424" : {
+	"description" : ""
+     },
+     "P2425" : {
+	"description" : ""
+     },
+     "P2426" : {
+	"description" : ""
+     },
+     "P2427" : {
+	"
```

Comment:
Trailing commas are no good in JSON.  http://stackoverflow.com/questions/201782/can-you-use-a-trailing-comma-in-a-json-object